index.Rmd

```diff
@@ -7,21 +7,21 @@ author: Hadarou Sare
+For this project, the most important criteria I  considered is the geological criteria for question of time. I chose and explored 9 sites as training sites and then selected among those sites the best ones as future landing/mining sites.
 
 # Materials and methods
 
-I choosed 9 sites as training sites (Si polar swath"I00831006"; Lyot crater"V00823007"; Cerberus fossae"V00825005";Echus chasma"I00839002"; Sinus Meridiana west"I00849005"; Syrtis Major"I00853002"; Ophir planum"I00864002"; Marte vallis"I00868006; Eastern Meridiani"I00836002").
-For each training site, I downloaded and used Raster data. Those data are Visible and infrared data in the “.TIFF” format and are combination of different band (9 0R 10 bands according to Mars Odysee website) and each band is showing data in a specific wavelength. I also downloaded Earth data from Landsat to show how I am applying what is done on Earth to Mars.After downloading Earth data, I  resized some of them so that all images will have the same size and I  then added them together to create a metadata containing all information from each site. For Mars, I don't need to resize any of the images because they all have the same size.I finally used the band ratio combination to determine the presence, abundance and distribution of water, water ice, ice or volatiles materials in each site I choose as training site and from there I was able to see which sites could be selected as the best future landing/mining sites.
+I chose 9 sites as training sites (Si polar swath"I00831006"; Lyot crater"V00823007"; Cerberus fossae"V00825005";Echus chasma"I00839002"; Sinus Meridiana west"I00849005"; Syrtis Major"I00853002"; Ophir planum"I00864002"; Marte vallis"I00868006; Eastern Meridiani"I00836002").
+For each training site, I downloaded and used Raster data. Those data are Visible and infrared data in the “.TIFF” format and are combination of different band (9 0R 10 bands according to Mars Odysee website) and each band is showing data in a specific wavelength. I also downloaded Earth data from Landsat to show how I am applying what is done on Earth to Mars. After downloading Earth data, I resized some of them so that all images will have the same size and I then added them together to create a metadata containing all information from each site. For Mars, I don't need to resize any of the images because they all have the same size.I finally used the band ratio combination to determine the presence, abundance and distribution of water, water ice, ice or volatiles materials in each site I chose as training site and from there I was able to see which sites could be selected as the best future landing/mining sites. 
```

Comment:
The last sentence is a run-on sentence that I could not figure out how to fix. So just adjust this sentence a bit to make it easier to read.